partner/app/main.py

```diff
@@ -89,6 +104,21 @@ async def read_tilesp(
     results: int = Query(1, example=2),
 ):
     """Endpoint for requests from Contile."""
+    unknown_params = []
+    for param in request.query_params:
+        if param not in ACCEPTED_QUERY_PARAMS:
+            unknown_params.append(param)
```

Comment:
We can express this with a list-comprehension:

```suggestion
    unknown_query_params = [
        param for param in request.query_params if param not in ACCEPTED_QUERY_PARAMS
    ]
```